### PR TITLE
Gazebo 10 sym links

### PR DESCRIPTION
**Pull request by nkoenig**

### CODE REVIEW

File: Aliases/gazebo10 [hunk 1]
```diff
@@ -0,0 +1 @@
+../gazebo8.rb
```
Comment (chapulina) [inline Aliases/gazebo10]: I think this should be 9, which needs to be changed to have different dependencies than 8

Comment (nkoenig) [reply Aliases/gazebo10]: pull request #349 is need first.

Comment (scpeters) [reply Aliases/gazebo10]: I just merged #349

Comment (nkoenig) [reply Aliases/gazebo10]: Update to point to gazebo9

**Comment by scpeters** [comment]
I don't see any changes in this branch

**Pull request merged**